rbo/__init__.py

```diff
@@ -1 +0,0 @@
-from .rbo import RankingSimilarity
```

Comment:
This change actively breaks the examples as shown in README.md. 

```python
>>> from rbo import RankingSimilarity  # this was previously working
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'RankingSimilarity' from 'rbo' (/Users/redacted/rbo/rbo/__init__.py)
>>> from rbo.rbo import RankingSimilarity. # this works now, but I think it's cumbersome
```

I prefer the old way because:
- it did not depend on calling "rbo" out twice as in "rbo.rbo".
- changing that breaks existing code
- hands us explicit control which names to expose. In this case, `RankingSimilarity` is the only publicly exposed name. Having this structure purposely hid `ProgressPrintOut`, `NoPrintOut` as they are implementation details.

Is there anything I'm missing?